### PR TITLE
ACA-111 - validate NoC request event count

### PR DESCRIPTION
**Pull request by smathangi**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ACA-111

### Change description ###

Validate uniqueness for NoC Request event

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by tomxelliott — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/managecase/service/noc/NoticeOfChangeQuestionsTest.java [hunk 1]
```diff
@@ -432,6 +433,22 @@ void shouldThrowErrorNoEventAvailable() {
                 .hasMessageContaining("No NoC events available for this case type");
         }
 
+        @Test
+        @DisplayName("Must return an error when multiple NOC Request events are available in the case for the user")
```
Comment (sullivangareth) [inline src/test/java/uk/gov/hmcts/reform/managecase/service/noc/NoticeOfChangeQuestionsTest.java:437]: Although its present in the integration test, should we test here for when there are no events available ?

**Review by sullivangareth — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/managecase/service/noc/NoticeOfChangeQuestionsTest.java (hunk 1 above)
Comment (sullivangareth) [reply src/test/java/uk/gov/hmcts/reform/managecase/service/noc/NoticeOfChangeQuestionsTest.java:437]: Oops- just seen the test above, please ignore

Comment (smathangi) [reply src/test/java/uk/gov/hmcts/reform/managecase/service/noc/NoticeOfChangeQuestionsTest.java:437]: It's already there :) line #425

**Review by mattnayler — APPROVED** [review]
I will approve 👍   but will also raise one extra question :

> should we update swagger or not as this is a low level failure caused bu bad config?

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request merged**